src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -44,12 +44,23 @@ class AttributeMetadata implements AttributeMetadataInterface
     /**
      * @var string|null
      *
+     * @deprecated since Symfony 5.1, to be removed in Symfony 6.0. Use $serializedNames instead
```

Comment:
`@deprecated since Symfony 5.1, use $serializedNames instead.`

Reply:
done

Reply:
This isn't necessary to deprecate this property by the way, because it's an internal one.